index.html

```diff
@@ -22,6 +22,7 @@ <h5>Keyboard shortcuts</h5>
               <li><span class="key">w</span>/<span class="key">s</span> move between users</li>
               <li><span class="key">a</span>/<span class="key">d</span> move between tasks</li>
               <li><span class="key">q</span> switch between task parts</li>
+              <small>Vim keys also work.</small>
```

Comment:
small in ul ist kein schönes HTML. Ich weiß auch nicht, ob jeder mit dem Begriff "vim keys" etwas anfangen kann. Wir könnten das als easter egg stehen lassen?

Reply:
dann lassen wir es weg, die die vim nutzen werden eh versuchen, ob vim shortcuts gehen.